daprdocs/content/en/concepts/dapr-services/scheduler.md

```diff
@@ -115,25 +115,85 @@ services:
+### List jobs
 
-#### Kubernetes Example
+```bash
+dapr scheduler list
+```
 
-Here's how to port forward and connect to the etcd instance:
+Example output:
+
+```bash
+NAME                                           BEGIN     COUNT  LAST TRIGGER
+actor/myactortype/actorid1/test1               -3.89s    1      2025-10-03T16:58:55Z
+actor/myactortype/actorid2/test2               -3.89s    1      2025-10-03T16:58:55Z
+app/test-scheduler/test1                       -3.89s    1      2025-10-03T16:58:55Z
+app/test-scheduler/test2                       -3.89s    1      2025-10-03T16:58:55Z
+activity/test-scheduler/xyz1::0::1             -888.8ms  0
+activity/test-scheduler/xyz2::0::1             -888.8ms  0
+workflow/test-scheduler/abc1/timer-0-TVIQGkvu  +50.0h    0
+workflow/test-scheduler/abc2/timer-0-OM2xqG9m  +50.0h    0
+```
 
-```shell
-kubectl port-forward svc/dapr-scheduler-server 2379:2379 -n dapr-system
+For more detail, use the wide output format:
+
+```bash
+dapr scheduler list -o wide
+```
+
+```yaml
+NAMESPACE  NAME                                           BEGIN                 EXPIRATION            SCHEDULE         DUE TIME                   TTL     REPEATS  COUNT  LAST TRIGGER
+default    actor/myactortype/actorid1/test1               2025-10-03T16:58:55Z                        @every 2h46m40s  2025-10-03T17:58:55+01:00          100      1      2025-10-03T16:58:55Z
+default    actor/myactortype/actorid2/test2               2025-10-03T16:58:55Z                        @every 2h46m40s  2025-10-03T17:58:55+01:00          100      1      2025-10-03T16:58:55Z
+default    app/test-scheduler/test1                       2025-10-03T16:58:55Z                        @every 100m      2025-10-03T17:58:55+01:00          1234     1      2025-10-03T16:58:55Z
+default    app/test-scheduler/test2                       2025-10-03T16:58:55Z  2025-10-03T19:45:35Z  @every 100m      2025-10-03T17:58:55+01:00  10000s  56788    1      2025-10-03T16:58:55Z
+default    activity/test-scheduler/xyz1::0::1             2025-10-03T16:58:58Z                                         0s                                          0
+default    activity/test-scheduler/xyz2::0::1             2025-10-03T16:58:58Z                                         0s                                          0
+default    workflow/test-scheduler/abc1/timer-0-TVIQGkvu  2025-10-05T18:58:58Z                                         2025-10-05T18:58:58Z                        0
+default    workflow/test-scheduler/abc2/timer-0-OM2xqG9m  2025-10-05T18:58:58Z                                         2025-10-05T18:58:58Z                        0
+```
+
+### Get job details
+
+```bash
+dapr scheduler get app/my-app/job1 -o yaml
+```
+
+### Delete jobs
+
+Delete one or more specific jobs:
+
+```bash
+dapr scheduler delete app/my-app/job1 actor/MyActor/123/reminder1
+```
+
+Bulk delete jobs with filters:
+
+```bash
+dapr scheduler delete-all all
```

Comment:
Was using a wildcard (`*`) considered here? so it could be:

```
dapr scheduler delete-all *
dapr scheduler delete-all app/*
dapr scheduler delete-all actor/*
etc
```